src/bare-metal/aps/uart.md

```diff
@@ -0,0 +1,5 @@
+# Let's write a UART driver
```

Comment:
The flow is a bit abrupt here: I suggest adding a slide first with this title. The slide would explain what UART is and why we should care :slightly_smiling_face:

Reply:
We don't particularly care, it's just a simple example to start with.